### DISCUSSION TITLE
Great course

### DISCUSSION THREAD

**Discussion by longtongster** [mrdbourke/pytorch-deep-learning#448] [General]
Daniel,
I want to thank for this great course. It was a great journey and you taught us with a lot of enthusiasm. I learned a lot. Please let me know if your going to some more advanced topics (e.g. distributed training, how to work with large datasets, callbacks, maybe train on AWS, build an object detector from scratch)

**Comment by mrdbourke** [comment]
Hi @longtongster,

Thank you for the kind words!

For now, I think the course is good for beginners, advanced topics are a bit harder to do because it minimizes the amount of people who can do them.

For example, with the fundamentals, you can start to branch into what you'd like to do.

However, I have had some ideas to add: 
- NLP with PyTorch
- Object detection

As these are quite common problems.

But no concrete plans yet.

**Reply by dmahdipour** [reply]
Thank you for your course.

**Comment by dmahdipour** [comment]
thanks alot.

**Comment by ssuhas76** [comment]
Hello @mrdbourke , Any update on the NLP for pytorch like torchtext? Will you be adding text specific parts like you had done it for Tensorflow
